### DISCUSSION TITLE
Can't consume object storage from Rook toolbox

### DISCUSSION THREAD

**Discussion by weeix** [rook/rook#12212] [General]
Hi, I'm following this document:

https://rook.io/docs/rook/v1.11/Storage-Configuration/Object-Storage-RGW/object-storage/#consume-the-object-storage

However, after connecting to the Rook toolbox:

1. I can't create the `~/.aws` directory (since `~` = `/`)

    ```
    bash-4.4$ mkdir ~/.aws
    mkdir: cannot create directory '//.aws': Permission denied
    ```

2. I can't run the `s5cmd` command

    ```
    bash-4.4$ s5cmd --help
    bash: s5cmd: command not found
    ```

Did I miss anything? Is there supposed to be some commands before the next sub-header?

![image](https://user-images.githubusercontent.com/6592791/237003012-5503e4f6-a11a-41df-947b-7077ebec506b.png)

Thanks.

**Comment by travisn** [comment]
Oh, thanks for reporting that the object doc is outdated. The s5cmd is included in the rook image, but since v1.10 the toolbox is instead using the ceph image by default, which does not have s5cmd. Please test the same instructions by using the rook image instead of the ceph image in the toolbox spec.

You'll need to create the toolbox spec based on the rook image as seen in the [v1.9 toolbox manifest](https://github.com/rook/rook/blob/release-1.9/deploy/examples/toolbox.yaml). Change the `image:` of the toolbox to match the image you're using in the operator. (The latest image is v1.11.5)

**Reply by weeix** [reply]
After changing the toolbox image from `quay.io/ceph/ceph:v17.2.6` to `rook/ceph:v1.11.5`, and adding the `export BUCKET_NAME=...` command inside the toolbox (in addition to the 4 environment variables), I was able to follow the document until the end.

Thank you.
